### ISSUE TITLE
bibtex parser errors

### ISSUE THREAD

**Issue by James-Yu** [tamuratak/latex-utensils#3]
There are some problems exist in the current bibtex pegjs. Here is an example as of commit f681b6e.

```
% Encoding: UTF-8

@Article{Lee_Robustvehiclerouting_2012,
  author    = {Lee, Chungmok and Lee, Kyungsik and Park, Sungsoo},
  title     = {Robust vehicle routing problem with deadlines and travel time/demand uncertainty},
  journal   = {Journal of the Operational Research Society},
  year      = {2012},
  volume    = {63},
  number    = {9},
  pages     = {1294--1306},
  doi       = {10.1057/jors.2011.136},
  file      = {:Lee_Robustvehiclerouting_2012 - Robust vehicle routing problem with deadlines and travel time_demand uncertainty.pdf:PDF},
  publisher = {Springer},
}
```

![image](https://user-images.githubusercontent.com/4210342/62823834-5a0a7980-bbc8-11e9-977b-9b78e55f2774.png)

It also had trouble dealing with journal abbr.

```
@Article{Yu_Autonomousvehiclelogistic_,
  author   = {J. J. Q. Yu and A. Y. S. Lam},
  title    = {Autonomous vehicle logistic system: joint routing and charging strategy},
  journal  = IEEE_J_ITS,
  year     = {2018},
  volume   = {19},
  number   = {7},
  pages    = {2175--2187},
  month    = jul,
  issn     = {1524-9050},
  doi      = {10.1109/TITS.2017.2766682},
  file     = {:Yu_AutonomousVehicleLogistic_ - Autonomous Vehicle Logistic System_ Joint Routing and Charging Strategy.pdf:PDF},
  keywords = {Autonomous vehicles, Logistics, Power system stability, Routing, Smart cities, Autonomous vehicle, logistics system, smart city., vehicle charging, vehicle routing},
}
```

![image](https://user-images.githubusercontent.com/4210342/62823843-832b0a00-bbc8-11e9-89dd-74b5b07133c7.png)

**Comment by tamuratak** [comment]
Thank you. I have fixed.

**Comment by James-Yu** [comment]
Good work. A last problem I found:
```
@Comment{jabref-meta: grouping:
0 AllEntriesGroup:;
1 StaticGroup:Markings\;2\;1\;\;\;\;;
2 StaticGroup:James:6\;2\;1\;\;\;\;;
}
```
yields
![image](https://user-images.githubusercontent.com/4210342/62829815-56acd780-bc34-11e9-8567-035246ace98c.png)
It is not standard bibtex, but it is generated by JabRef, which is a really popular solution among the LaTeX community.

**Comment by tamuratak** [comment]
Thank you. I have fixed with fb2e4104ca1381c8ec2fb4c879567f291d23159f.

**Comment by James-Yu** [comment]
The previous example is handled. Yet unfortunately the error persists when combining the previous ones together:
```
% Encoding: UTF-8

@Article{Lee_Robustvehiclerouting_2012,
  author    = {Lee, Chungmok and Lee, Kyungsik and Park, Sungsoo},
}

@Comment{jabref-meta: databaseType:bibtex;}

@Comment{jabref-meta: grouping:
0 AllEntriesGroup:;
1 StaticGroup:Markings\;2\;1\;\;\;\;;
2 StaticGroup:James:6\;2\;1\;\;\;\;;
}
```
yields
![image](https://user-images.githubusercontent.com/4210342/62830281-16068b80-bc3f-11e9-9a84-e3a07868ad8d.png)

**Comment by tamuratak** [comment]
Fixed with 0ae7b10d173f62d2588dd47cc2b06e2281e884f1.
